composer.json

```diff
@@ -9,14 +9,15 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
-        "illuminate/support": "~5.0",
+        "php": ">=5.6.4",
```

Comment:
this package shipped to work with  5.3 or higher , so dont think this is needed.